lib/app/utils/index_builder.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
+
+require 'json'
+
+module Inferno
+  class IndexBuilder
+    # @param folder [String] the path of the package to index
+    def initialize(folder)
+      raise ArgumentError, "'#{folder}' is not a directory" unless Dir.exist?(folder)
+
+      package_folder = File.join(folder, 'package')
+      (folder = package_folder) if Dir.exist?(package_folder)
```

Comment:
These parens aren't necessary.
```suggestion
      folder = package_folder if Dir.exist?(package_folder)
```